en/gps_compass/rtk_gps_trimble_mb_two.md

```diff
@@ -49,6 +49,8 @@ The pins on the 28-pin connector are numbered as shown below:
 
 ## Configuration
 
+First set the GPS protocol to Trimble ([GPS_x_PROTOCOL=3](../advanced_config/parameter_reference.md#GPS_1_PROTOCOL)).
```

Comment:
@bkueng Thanks, looks great, merged.

I added this point in the Trimble MB2 config because it is where you are likely to run into problems. I don't think we have stand alone pages for any other non-ublox GPS right - ie. all of the peers of this one are ublox right? http://docs.px4.io/master/en/gps_compass/rtk_gps_trimble_mb_two.html

Reply:
Correct, the vast majority is on u-blox